src/index.css

```diff
@@ -88,6 +94,43 @@ main {
   margin-top: 20px;
 }
 
+.single-article-topic {
+  font-style: italic;
+  font-weight: 500;
+}
+
+.single-article-votes {
+  margin-top: 10px;
+  color: rgb(128, 5, 128);
+  font-weight: 500;
+}
+
+.upvote {
+  display: inline-block;
+  width: 0;
+  height: 0;
+  padding: 0;
+  border: 0;
+  outline: none;
+  border-left: 20px solid transparent;
+  border-right: 20px solid transparent;
+  border-bottom: 20px solid green;
+  cursor: pointer;
+}
+
+.downvote {
+  display: inline-block;
+  width: 0;
+  height: 0;
+  padding: 0;
+  border: 0;
+  outline: none;
+  border-left: 20px solid transparent;
+  border-right: 20px solid transparent;
+  border-top: 20px solid red;
+  cursor: pointer;
+}
```

Comment:
could refactor the CSS here - with a group class `vote` you can have `display, width, height, padding, border, outline, border-left, border-right, cursor` as both have the same. Then you can do `border-top` differently for `upvote` and `downvote`. With the quality of your CSS, you could probably tackle Tailwind. Once you start Tailwind though, there is no going back..